### ISSUE TITLE
Problem with Mangazuki

### ISSUE THREAD

**Issue by molitar** [fmd-project-team/FMD#482]
If I go directly to the site I can find the manhwa.

https://www.mangazuki.online/manga/hahris-lumpy-star/

But when I try to find this on any of the MangaZuki sites through FMD it does not find it.  As their is 36 chapters but all other sites only have 33 chapters.

**Comment by SDXC** [comment]
The url of that title changed. You can update the manga list so the url gets fixed or just wait until an updated database has been uploaded.

**Comment by molitar** [comment]
Yeah I tried to do update manga list. Did not work still shows as 0 NA.  So will wait for updated Database I guess.

**Comment by molitar** [comment]
Ok Mangazuki.online and Mangazuki.site both of these sites are unable to download manga list.  Unable to retrieve the catalog at all.

**Comment by SDXC** [comment]
They probably changed their template.

**Comment by AbidZakaria** [comment]
> If I go directly to the site I can find the manhwa.
> 
> https://www.mangazuki.online/manga/hahris-lumpy-star/
> 
> But when I try to find this on any of the MangaZuki sites through FMD it does not find it. As their is 36 chapters but all other sites only have 33 chapters.

![image](https://user-images.githubusercontent.com/19544118/64677793-7db03080-d4a2-11e9-8ab8-d9bd69c314ca.png)

working fine on me

**Comment by AbidZakaria** [comment]
> Ok Mangazuki.online and Mangazuki.site both of these sites are unable to download manga list. Unable to retrieve the catalog at all.
> 
> Now on Mangazuki.site for some reason it appears FMD is unable to connect because the downloaded URLS from the server is correct.
> 
> https://www.mangazuki.site/manga/the-bathtub/
> 
> It shows correct in both FMD and the URL and if I load the URL it opens fine. But FMD fines no chapters.

![image](https://user-images.githubusercontent.com/19544118/64677904-ac2e0b80-d4a2-11e9-923d-a6911cc940d9.png)
